docs/specs/config.yml

```diff
@@ -313,8 +317,26 @@ inputs:
 outputs:
   docs/a.json:
   build.log: |
-    ["error","invalid-redirection","The 'docs/TOC.md' shouldn't belong to redirections since it's a TableOfContents","docfx.yml"]
-    ["error","redirection-conflict","The 'docs/a.md' appears twice or more in the redirection mappings","docfx.yml"]
+    ["error","invalid-redirection","The 'docs/TOC.md' shouldn't belong to redirections since it's a TableOfContents","redirection.yml"]
+    ["error","redirection-conflict","The 'docs/a.md' appears twice or more in the redirection mappings","redirection.yml"]
+---
+# Folder level redirection.yml or redirection.json files
+inputs:
+  docfx.yml:
+  redirection.yml: |
+    redirections:
+      docs/a.md: /a
+  docs/folder-1/redirection.json: |
+      { "redirections" : { "docs/folder-1/a.md": "/b" } }
```

Comment:
Feels rather confusing that  `docs/folder-1/a.md` is relative to docset root instead of the current folder...